src/gui/plugins/entity_tree/EntityTree.cc

```diff
@@ -434,5 +434,5 @@ bool EntityTree::eventFilter(QObject *_obj, QEvent *_event)
 }
 
 // Register this plugin
-IGNITION_ADD_PLUGIN(ignition::gazebo::EntityTree,
+IGNITION_ADD_PLUGIN(EntityTree,
                     ignition::gui::Plugin)
```

Comment:
Interesting that it didn't catch some of these `ignition::`. Or are we keeping it because the compiler can't tell `ignition::gui` from `ignition::gazebo::gui`?

Reply:
Yep! It 💥 otherwise.

Most cases where it doesn't catch is from me putting in an explicit escape hatch (: